CriteoMoPubAdapter/Sources/CriteoMoPubAdapter/CRCustomEventHelper.m

```diff
@@ -24,28 +24,15 @@
 
 @implementation CRCustomEventHelper
 
-+ (BOOL)checkValidInfo:(NSDictionary *)info {
-  return [self checkValidInfo:info withError:nil];
-}
-
-+ (BOOL)checkValidInfo:(NSDictionary *)eventInfo withError:(NSError **)error {
++ (BOOL)checkValidInfo:(NSDictionary *)eventInfo {
   NSArray<NSString *> *expectedKeys = @[ kCRCustomEventHelperCpId, kCRCustomEventHelperAdUnitId ];
-  NSMutableString *errorMsg = [[NSMutableString alloc] init];
   BOOL isValid = YES;
   for (NSString *key in expectedKeys) {
     NSString *value = eventInfo[key];
     if (![value isKindOfClass:NSString.class] || (value.length == 0)) {
       isValid = NO;
-      NSString *str =
```

Comment:
OPT: I guess you removed that on purpose, but aren't these cases you might want to diagnose at some point, i.e. worth logging?